osquery/extensions/interface.cpp

```diff
@@ -64,6 +67,23 @@ void ExtensionHandler::shutdown() {
   Initializer::requestShutdown(EXIT_SUCCESS);
 }
 
+/**
+ * @brief Updates the Thrift server output to be VLOG
+ *
+ * On Windows, the thrift server will output to stdout, which displays
+ * messages to the user on exiting the client. This function is used
+ * instead of the default output for thrift.
+ *
+ * @param msg The text to be logged
+ */
+void thriftLoggingOutput(const char* msg) {
+  VLOG(1) << msg;
+}
+
+ExtensionManagerHandler::ExtensionManagerHandler() {
+  GlobalOutput.setOutputFunction(thriftLoggingOutput);
```

Comment:
This is awesome, is there any tear down we need to handle (as in when the process exits)?